src/browser/modules/Sidebar/DocumentItems.tsx

```diff
@@ -119,14 +124,28 @@ export const DocumentItems = ({
 }
 
 type CommandItemProps = Command & { executeCommand: (cmd: string) => void }
-const CommandItem = ({ name, command, executeCommand }: CommandItemProps) => (
-  <StyledCommandListItem onClick={() => executeCommand(command)}>
-    <StyledCommandNamePair>
-      <StyledName> {name} </StyledName>
-      <StyledCommand> {command} </StyledCommand>
-    </StyledCommandNamePair>
-  </StyledCommandListItem>
-)
+const CommandItem = ({ name, command, executeCommand }: CommandItemProps) => {
+  const [showRunButton, setShowRunButton] = useState(false)
```

Comment:
We has this behaviour before as well but it was done with :hover in css right?

Reply:
Kinda nice to have it be done in pure CSS but I see how it can be impractical in with the new way of doing things :+1: